packages/ui/src/fields/Relationship/index.tsx

```diff
@@ -457,26 +457,25 @@ const RelationshipFieldComponent: RelationshipFieldClientComponent = (props) =>
         i18n,
       })
 
+      const currentValue = valueRef.current
+      const docId = args.doc.id
```

Comment:
NBD, but try to match the casing patterns for acronyms. This should be docID. Which would be consistent with things like payload.findByID. This comes from JS core with naming like date.toUTCString().